irods/parallel.py

```diff
@@ -324,7 +346,10 @@ def _io_multipart_threaded(operation_ , dataObj_and_IO, replica_token, hier_str,
         if Io is None:
             Io = session.data_objects.open( D.path, Operation.data_object_mode(initial_open = False),
                                             create = False, finalize_on_close = False,
-                                            **{kw.RESC_HIER_STR_KW: hier_str, kw.REPLICA_TOKEN_KW: replica_token} )
+                                            **{ kw.NUM_THREADS_KW: str(num_threads),
+                                                kw.DATA_SIZE_KW: str(total_size),
+                                                kw.RESC_HIER_STR_KW: hier_str,
+                                                kw.REPLICA_TOKEN_KW: replica_token })
         mgr.add_io( Io )
         if File is None: File = gen_file_handle()
         futures.append(executor.submit( _io_part, Io, r, File, Operation, mgr, str(counter), queueObject))
```

Comment:
consider replacing `r` with a word that expresses what it is.

Reply:
Will do.